samples/java_layout/src/main/java/io/customer/android/sample/java_layout/ui/settings/SettingsActivity.java

```diff
@@ -118,6 +115,7 @@ private void setupViews() {
             ClipboardManager clipboard = (ClipboardManager) getSystemService(Context.CLIPBOARD_SERVICE);
             ClipData clip = ClipData.newPlainText(getString(R.string.device_token), deviceToken);
             clipboard.setPrimaryClip(clip);
+            Toast.makeText(this, R.string.token_copied, Toast.LENGTH_SHORT).show();
```

Comment:
I think Android OS does pop-up info on copied text which is generally enough, but that might only be on Pixel phones. So good to have toast that is consistent for all phones.

Reply:
Yeah this is pixel only feature i suppose 😄